lib/rubocop/cop/rails/unknown_env.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop checks that environments called with Rails.env predicates
```

Comment:
It may be better to enclose `Rails.env` in backquotes.
BTW, This pragmatic cop is awesome. Thanks always.

Reply:
> It may be better to enclose Rails.env in backquotes.

Good catch, thanks!

> BTW, This pragmatic cop is awesome. Thanks always.
 

:smile_cat:

Reply:
Thanks for the update. Moreover, it seems necessary to regenerate the document with `rake generate_cops_documentation`.
https://travis-ci.org/bbatsov/rubocop/jobs/280723348#L590